Resources/doc/setup.md

```diff
@@ -86,4 +86,35 @@ _ecentria_rest_bundle:
 NelmioApiDocBundle:
     resource: "@NelmioApiDocBundle/Resources/config/routing.yml"
     prefix:   /api/doc
-```
\ No newline at end of file
+```
+
+E: (Optional) Setup multiple entity managers
+--------------------------------
+
+It's a specific way of setting up the EM (entity manager) so that the EcentriaRestBundle can intelligently handle them.
+EM would be automatically determined from an entity. The *prefix* and *dir* config options below are very important in determining the EM associated with the entity.
+
+Modify app/config/config.yml and add the following:
```

Comment:
"Update your app/config/config.yml with something similar to the example below:"